src/modules/infobox/components/styling/FeatureStyleEdit.vue

```diff
@@ -23,16 +136,19 @@
                 v-model="description"
                 :readonly="readOnly"
                 data-cy="drawing-style-feature-description"
-                class="form-control"
-                rows="2"
+                class="feature-description form-control"
+                :class="{
+                    'form-control-plaintext': readOnly,
+                }"
             ></textarea>
             <div v-if="isFeatureLine">
                 <font-awesome-icon :icon="['far', 'square']" />
-                {{ lengthRounded }} {{ length > 100 ? 'km' : 'm' }}
+                {{ roundValueIfGreaterThan(length, 100, 1000) }} {{ length > 100 ? 'km' : 'm' }}
             </div>
             <div v-if="isFeaturePolygon">
                 <font-awesome-icon :icon="['far', 'square']" class="bg-secondary text-secondary" />
-                {{ areaRounded }} {{ area > 10000 ? 'km' : 'm' }}<sup>2</sup>
+                {{ roundValueIfGreaterThan(area, 1000, 100000) }} {{ area > 10000 ? 'km' : 'm'
+                }}<sup>2</sup>
```

Comment:
`roundValueIfGreaterThan` could be integrated in the `length` and `area` computed property, might be more efficient and would be more readable in the template.